homework_3/src/main/java/com/epam/spring/service/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.spring.service.mapper;
+
+import com.epam.spring.controller.dto.CategoryDto;
+import com.epam.spring.service.model.Category;
+import org.mapstruct.Mapper;
+
+@Mapper
+public interface CategoryMapper {
+
+    CategoryDto mapCategoryDto(Category category);
```

Comment:
Hi. Could you please reconsider naming for the Mapper's methods.
For instance, we might use something like this
1. Model -> Dto: mapModelToDto
2. Dto -> Model: mapDtoToModel

Please apply new naming for all Mappers.
Thank you